### DISCUSSION TITLE
RFC: OpenXLA Project Governance

### DISCUSSION THREAD

**Discussion by theadactyl** [openxla/community#29] [Project Discussions]
I've published a proposal for OpenXLA project governance as a [PR](https://github.com/openxla/community/pull/28) in the community repo. 

For a high-level overview, take a look at the [slides](https://github.com/openxla/community/blob/main/meetings/2022-12-13-community-meeting/OpenXLA%20Governance%20Proposal%20v1.pdf) I presented at the 12/13/2022 community meeting ([video](https://youtu.be/xLZRgGozwBY)), where you'll find  to get an overview of the goals and principles we are hoping to achieve with this model.

This document is a draft (hence many TODOs), and the PR is open for community review and feedback until Jan 13, 2023. We plan to discuss any outstanding comments or revisions in the OpenXLA community meeting on Jan 17, 2023!

**Comment by cloudhan** [comment]
I think I just found a time traveler.
